### DISCUSSION TITLE
DMS question

### DISCUSSION THREAD

**Discussion by MarcosGraficon** [OCA/odoo-community.org#144] [Q&A]
Hey, im trying to save the attatchemts from odoo to an external NAS directly, but im not able to do (i'm a junior alone developer in my company), and I saw the DMS app, is there a way to store the files in a diferent server from this app? or is better to keep searching how to make it on my own? Ty guys

**Comment by vdewulf** [comment]
Hello,
I'm have no technical skills, this answer might be off, but I think this video from 2023 OCA Days could help answer your question: 
https://www.youtube.com/watch?v=7EQsw2pCCFA
Have a nice day!
